docs/topics/native-and-cross-platform.md

```diff
@@ -0,0 +1,130 @@
+
+These are just a few of the key advantages of cross-platform app development. You can learn more about its benefits and use cases from global companies in our article about [cross-platform mobile development](cross-platform-mobile-development.md). As for the challenges of the approach – we’ll discuss those in the following section.
+
+### Debugging some popular myths about cross-platform app development 
+
+Multiplatform technology is constantly evolving. Some cross-platform development frameworks like [Kotlin Multiplatform Mobile](https://kotlinlang.org/lp/mobile/) provide the benefits of building both cross-platform and native apps and remove the limitations that are commonly associated with the approach.
+
+#### 1. Cross-platform apps provide poorer performance than native apps.
+
+Poor performance was long considered to be one of the main disadvantages of multiplatform applications. However, the performance and quality of your product largely depend on the tools you use to build the app. The latest cross-platform frameworks provide all the tools necessary to develop apps with a native-like user experience.
+
+By using different compiler backends, [Kotlin](https://kotlinlang.org/docs/multiplatform.html#code-sharing-between-platforms) is compiled to platform formats – JVM bytecode for Android and native binaries for iOS. As a result, the performance of your shared code is the same as if you write them natively.
+
+#### 2. Cross-platform frameworks are unsafe.
+
+There’s a common misconception that native apps are much more secure and reliable. However, modern cross-platform development tools allow developers to build safe apps that guarantee reliable data protection. Mobile engineers just need to [take additional measures to make their apps secure](https://appstronauts.co/blog/are-cross-platform-apps-as-fast-and-secure-as-native-apps/#:~:text=Unsecurity%20of%20cross%2Dplatform%20apps,a%20cross%2Dplatform%20app%27s%20code.).
+
+#### 3. Cross-platform apps don’t have access to all native functions of mobile devices.
+
+It is true that not all cross-platform frameworks allow you to build apps with full access to the device’s features. Nevertheless, some modern multiplatform frameworks can help you overcome this challenge. For example, Kotlin Multiplatform Mobile gives easy access to Android and iOS SDKs. It provides a [Kotlin mechanism of expected and actual declarations](multiplatform-connect-to-apis.md) to help you access the device’s capabilities and features.  
+
+#### 4. It can often be difficult to manage cross-platform projects.
+
+In fact, it’s the opposite. Cross-platform solutions help you more effectively manage resources. Your development teams can learn how to write and reuse shared code. Android and iOS developers can achieve high efficiency and transparency by interacting and sharing knowledge.
+
+## 6 key aspects to help you choose between cross-platform app development and the native approach
```

Comment:
It's better to leave text instead of digit here: 
* it will be easier to read
* it looks like a part of a list before but it actually isn't